Makefile

```diff
@@ -128,6 +128,7 @@ endef
 
 # Launches Lumigator in 'development' mode (all services running locally, code mounted in)
 local-up: config-generate-env
+	git config --unset-all core.hooksPath
 	uv run pre-commit install
```

Comment:
when running `npm install` which runs the prepare script in https://github.com/mozilla-ai/lumigator/blob/2574e8907766cba101eb03ac11c3eadc465c62d6/lumigator/frontend/package.json#L18

husky sets the `core.hooksPath` to `frontend/.husky` which breaks whatever checks `uv pre-commit` does when running `make local-up`

I was getting this error
```sh
uv run pre-commit install
[ERROR] Cowardly refusing to install hooks with core.hooksPath set.
hint: git config --unset-all core.hooksPath
make: *** [local-up] Error 1
```

so this is a workaround to unset `core.hooksPath` before running `uv run pre-commit install` to make things work, and `core.hooksPath` then later gets set again whenever `npm install` is called